quipucords/api/aggregate_report/model.py

```diff
@@ -104,11 +134,7 @@ def reformat_aggregate_report_to_dict(aggregated: AggregateReport) -> dict:
 
 
 def get_aggregate_report_by_report_id(report_id: int) -> dict | None:
```

Comment:
I think this helper made sense when AggregateReport wasn't a model. Now this function should be deprecated and `get_object_or_404` from `from `rest_framework.generics` should be used instead.